server/data/projects.js

```diff
@@ -93,7 +97,57 @@ const createProject = async (projectObjParam, user) => {
 	return createdProject;
 };
 
+const updateProject = async (projectObjParam, id, user) => {
+	let projectCheck = await getProjectById(id);
+	ownercheck = checkuseraccess(user, projectCheck.owner);
```

Comment:
you are passing the objectId string and objectId. and in the helper function, you have done a `===` comparison, does that work?